rules-reviewed/openshift/session.windup.xml

```diff
@@ -26,37 +26,28 @@
             <perform>
                 <hint title="HTTP session replication (distributable web.xml)" effort="3" category-id="cloud-mandatory">
                     <message>
-                    <![CDATA[
-                       Session replication ensures that client sessions are not disrupted by failovers of nodes in a cluster.
-                       Each node in the cluster shares information about ongoing sessions and can take over sessions if a node disappears.
-                       The `<distributable/>` tag inside the `<web-app>` tag of application’s `web.xml` descriptor file enables the application's sessions clustering [1].
+                    Session replication ensures that client sessions are not disrupted by node failure. Each node in the cluster shares information about ongoing sessions and can take over sessions if another node disappears. In a cloud environment, however, data in the memory of a running container can be wiped out by a restart.
 
-                       In a cloud environment it has to be considered that the data in the memory of a running container can be wiped out by a restart (triggered by code deploy, config change, or the execution environment relocating the process to a different physical location)[2].
+                    Recommendations:
 
-                       Different approaches can be followed:
+                    Review the session replication usage and ensure that it is configured properly.
```

Comment:
if the section is "Recommendations:" , wdyt on having lines 33, 35,27 as an ordered list or bullet list ?

Reply:
Agree